webdriverio_taf/support/commands.js

```diff
@@ -0,0 +1,81 @@
+import { step } from '../support/helpers.js'
+
+export default () => {
+  browser.addCommand('scrollToElement', async function (targetElement) {
+    await step('Scroll to element', async () => {
+      const element = typeof targetElement === 'function'
+        ? await targetElement()
+        : await targetElement
+      const location = await element.getLocation()
+
+      await browser.execute((x, y) => {
+        window.scrollTo({
+          left: x - window.innerWidth / 2,
+          top: y - window.innerHeight / 2,
+        })
+      }, location.x, location.y)
+    })
+  })
```

Comment:
Let's apply same approach to steps everywhere:
```suggestion
  browser.addCommand('scrollToElement', function (targetElement) {
    return step('Scroll to element', async () => {
      const element = typeof targetElement === 'function'
        ? await targetElement()
        : await targetElement
      const location = await element.getLocation()

      return browser.execute((x, y) => {
        window.scrollTo({
          left: x - window.innerWidth / 2,
          top: y - window.innerHeight / 2,
        })
      }, location.x, location.y)
    })
  })
```

Reply:
Done. Applied to all custom commands, and also to steps in `webdriverio_taf\steps\` and helpers in `webdriverio_taf\support\helpers.js`